### DISCUSSION TITLE
so-network-setup freezes at 98% during heavy node install

### DISCUSSION THREAD

**Discussion by radman13666** [Security-Onion-Solutions/securityonion#3998] [General]
Hello,

I have a DELL PowerEdge R740xd server, with Ubuntu 18.04 installed, that I would like to install SO 2.3.40 as a heavy node.

I am using the Git installation method: https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos. The ISO method failed for me because my boot media (a USB flash drive) was "unsupported" for some reason.

I setup the installation as I require it, but it freezes at 98% as shown in the screenshot below. It has been like this for 3 days.

![image](https://user-images.githubusercontent.com/19785740/116041122-83598a80-a675-11eb-967e-69b128e7d12d.png)

What should I be looking out for to correct this? Any help will be appreciated. Thanks.

**Comment by weslambert** [comment]
Try checking the log at `/root/sosetup.log` for clues

**Reply by radman13666** [reply]
I've checked and I've found that the reason is because I configured IPv6 network settings on the sensor, and that from trying it again, it shows that the setup erases IPv6 network settings and the /etc/hosts file, yet it has captured the IPv6 address of the manager node.

So how can I make it read the IPv4 address instead when setting up? Is there a file I can erase so that it stops reading the IPv6 address of the manager node? or is there a way I can stop if from erasing the /etc/hosts file so that it can detect the IPv4 address instead?

**Reply by lruppert** [reply]
When mine froze at 98% it was because the key for salt stack was missing. #4486
